♻️ refactor: oauth 오류메세지 리팩터링-> 커스텀+로깅 으로 변경

SongMihyun opened the pull request:
- oauth 오류코드 리팩토링
- extend tag - 안쓰는 api -> x 로 태그 변경

### 📌 관련 이슈
- 이슈 번호: # (연관된 이슈 번호를 적어주세요)

### 📝 작업 내용
- [x]oauth 오류코드 리팩토링
- [x]extend tag - 안쓰는 api -> x 로 태그 변경
- [ ] 기타 필요한 설명

### 📋 체크리스트
- [ ] 코드에 오류가 없는지 확인했습니다.
- [ ] 문서가 업데이트되었습니다.
- [ ] 테스트를 통과했습니다.

### 🔗 참고 자료
- 추가로 참고할 자료가 있다면 링크를 추가하세요.

### 아래 # 뒤에 PR머지와 함께 Close할 이슈번호를 입력해주세요.
closes #

Commit:
♻️ refactor: oauth 오류메세지 리팩터링-> 커스텀+로깅 으로 변경

- oauth 오류코드 리팩토링
- extend tag - 안쓰는 api -> x 로 태그 변경

Commit:
test.sh

Meoyoug commented on src/expert/views.py:
```diff
@@ -179,6 +182,7 @@ def put(self, request, *args, **kwargs):
         return self.update(request, *args, **kwargs)
 
 
+@extend_schema(tags=["X"])
```
이거 사용할텐데 왜 x인가요?

Meoyoug commented on src/expert/views.py:
```diff
@@ -86,6 +88,7 @@ def get(self, request, *args, **kwargs):
         return Response(serializer.data)
 
 
+@extend_schema(tags=["X"])
```
얘도 사용할텐데요

Meoyoug commented on src/expert/views.py:
```diff
@@ -21,6 +21,7 @@
 )
 
 
+@extend_schema(tags=["X"])
```
전문가쪽 왜 다 X되어있는지 설명좀요

Commit:
test.sh

Meoyoug commented on src/common/exceptions.py:
커스텀 API Exception을 만들어 둔건 좋은데 Exception 네이밍도 필요해보이고 사용된 부분은 안보이네요??

Meoyoug reviewed (commented):
리뷰내용 해결 후에 다시 커밋 푸시하시고 디엠주세요

Commit:
🐛 fix: 서비스목록, 지역목록 세부내역 변경
💡 chore: oauth 오류메세지 수정/extend 메세지 수정

-오류 코드 정리 리팩토링
-서비스, 지역 관련 세부명칭 변경 및 choices 변경
-extend 표기 변경

Meoyoug reviewed (approved):
고생하셨습니다.
